Jenkinsfile

```diff
@@ -55,7 +55,7 @@ pipeline {
         stage('Publish final version images') {
             when {
                 expression {
-                    return isVersionTag(readCurrentTag())
+                    return isVersionTag(${TAG_NAME})
```

Comment:
It does not work, what do you think about this:

```
def String readCurrentTag() {
    return sh(returnStdout: true, script: "echo ${TAG_NAME}").trim()
}
```

Reply:
yep, lemme try it

Reply:
marking this as solved